### PR TITLE
Error handle & fix 5 hour delay

### PR DESCRIPTION
**Pull request by Tristie**
This fixes the 5 hour delay and also makes it so broken instances don't break the entire site

### CODE REVIEW

File: playground/loggo/index.html [hunk 1]
```diff
@@ -279,10 +279,11 @@ <h2 id="feed-heading">Feed <button id="refresh-sign" class="secondary span">Refr
             )
         );
         for (let instance of instances) {
+            if (!Array.isArray(instance?.feed)) continue
             for (let post of instance.feed) {
                 const [status, username, timestamp, gif] = post;
                 const isGifValid = /^(tode|berd|bot)$/.test(gif);
-                if (new Date(timestamp) > new Date()) {
+                if (new Date(new Date(post.date - 18000000)) > new Date()) {
```
Comment (TodePond) [inline playground/loggo/index.html]: What's the reasoning for this magic number?

**Review by TodePond — APPROVED** [review]

File: playground/loggo/index.html (hunk 1 above)
Comment (Tristie) [reply playground/loggo/index.html]: 5 hours in milliseconds (Login appears to save statuses with a date 5 hours in advance)

Comment (TodePond) [reply playground/loggo/index.html]: ok cool! maybe worth a comment :)

Comment (Tristie) [reply playground/loggo/index.html]: Updated 👍

**Comment by Svenlaa** [comment]
i never noticed the 5 hour delay, just looked into it.
the delay is based on timezone. at utc+1 (my location) there is no delay.
setting my phone to utc+10 adds a delay

**Comment by Tristie** [comment]
Oh I should have further tested time zones since I considered that but the API returns UTC so I thought all was fine. Now that I know the actual error, I'll try to PR around to fix it without a hard-coded offset

**Pull request merged**